news/212.bugfix

```diff
@@ -0,0 +1,2 @@
+Fixed/ Revamped cluttered action buttons in Theming Control Panel
```

Comment:
```suggestion
The action buttons in the theming control panel have been improved
```

Reply:
@ale-rt sorry' didn't noticed this one ..
I'll edit this one directly! 😑